backend/controllers/auth.js

```diff
@@ -1,20 +1,32 @@
 import { HttpError } from "../helpers/HttpError.js";
 import { createApiResponse } from "../helpers/createApiResponse.js";
 import { prisma } from "../lib/db.js";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcrypt";
 
 const COOKIE_MAX_AGE_MILISECONDS = 60 * 60 * 1000;
+const { JWT_SECRET } = process.env;
 
 const register = async (req, res, next) => {
   try {
     const { username, password } = req.body;
+
+    const hashPassword = await bcrypt.hash(password, 10);
+
     const user = await prisma.user.create({
       data: {
-        password,
+        password: hashPassword,
         username,
       },
     });
+
+    const payload = {
+      id: user.id,
+    };
+    const token = jwt.sign(payload, JWT_SECRET, { expiresIn: "23h" });
```

Comment:
I see that cookie max age time, and expiration time of the token, differ significantly. Please adjust them to be equal, and move the jwt expiration time in a const as well